src/components/Garden/Preferences/EVMExecutor.js

```diff
@@ -307,31 +305,15 @@ function EVMExecutor({ evmcrispr }) {
         </>
       )}
       {terminalMode && (
-        <>
-          <Box
-            css={`
-              z-index: 1;
-            `}
-          >
-            <AceEditor
-              width="100%"
-              mode="jade"
-              value={code}
-              onChange={setCode}
-              fontSize={14}
-              showPrintMargin={false}
-              showGutter={false}
-              highlightActiveLine
-              setOptions={{
-                enableBasicAutocompletion: true,
-                enableLiveAutocompletion: true,
-                enableSnippets: true,
-                showLineNumbers: true,
-                tabSize: 2,
-              }}
-            />
-          </Box>
-        </>
+        <TextInput
+          onChange={setCode}
```

Comment:
We need to create a handler for this changebecause the argument will be the event object.